feder/institutions/viewsets.py

```diff
@@ -19,15 +23,61 @@ class Meta:
         fields = ["name", "tags", "jst", "regon"]
 
 
+class InstitutionCSVRenderer(PaginatedCSVRenderer):
+    header = [
+        "pk",
+        "name",
+        "email",
+        "jst",
+        "jst_name",
+        "jst_voivodeship",
+        "jst_category",
+        "created",
+        "modified",
+        "regon",
+        "self",
+        "slug",
+        "tags.0",
+        "url",
+    ]
+    labels = {
+        "pk": _("Primary key"),
+        "name": _("Name"),
+        "email": _("Email of institution"),
+        "jst": _("JST id"),
+        "jst_name": _("JST name"),
+        "jst_category": _("JST category"),
+        "jst_voivodeship": _("JST voivodeship"),
+        "created": _("Created"),
+        "modified": _("Modified"),
+        "regon": _("REGON number"),
+        "self": _("Parent institutions"),
+        "slug": _("Slug"),
+        "tags.0": _("Tag"),
+        "url": _("URL"),
+    }
+
+
+class InstitutionPaginator(PageNumberPagination):
+    max_page_size = (
```

Comment:
How will this impact on environmental stability?

Reply:
Well, for large page sizes it would take much time and resources to respond, so I think it would be better to restrict access to REST API only to logged in users. What do You think?

Reply:
In my opinion, the correct export of tags is also important to properly manage institutions. Maybe just separate it with commas or a semicolon to return everything?

Reply:
We tries to open the data as much as possible. Why will the response take a lot of resources? Everything seems to be generated in the stream.

I do not understand why the API for institutions should be closed, which will cause us problems in the future, if someone had to download the data or use in another system. Do I remember correctly that SJJ uses the institutional REST API of Fedrowanie? So, are you going to block the REST API and then start user sync with SJJ?

Reply:
I see that stream usage require other rendered ( https://github.com/mjumbewu/django-rest-framework-csv/blob/master/rest_framework_csv/renderers.py#L196 ).